benchmark/sparse/linear_regression/keras_tf_model.py

```diff
@@ -0,0 +1,47 @@
+"""
+Linear Regression model with sparse synthetic data for Keras
+"""
+
+import time
+
+import numpy as np
+from keras.layers import Dense, Input
+from keras.models import Sequential
+from keras.optimizers import SGD
+from scipy.sparse import csc_matrix
+
+# Fix random seed for reproducibility
+np.random.seed(7)
+
+start = time.time()
```

Comment:
It's better to only time the training time. As data generation code is different in pure mxnet (extra step to crate Iterators)

Reply:
Good point, fixed.